public_html/wp-content/mu-plugins/blocks/assets/src/speakers/edit.js

```diff
@@ -1,61 +1,26 @@
 /**
  * External dependencies
  */
-import { isUndefined, pickBy, split } from 'lodash';
+import { orderBy, intersection, split } from 'lodash';
```

Comment:
Are these still needed in this file?

Reply:
Nope, feel free to remove.

Reply:
Fixed in 9a7973a6dbb926fa6ba894e75f7a0bdb76f97d5b